Makefile-deps.mk

```diff
@@ -111,8 +111,12 @@ mpi-operator-crd: ## Copy the CRDs from the mpi-operator to the dep-crds directo
 KF_TRAINING_ROOT = $(shell $(GO_CMD) list -m -mod=readonly -f "{{.Dir}}" github.com/kubeflow/training-operator)
 .PHONY: kf-training-operator-crd
 kf-training-operator-crd: ## Copy the CRDs from the training-operator to the dep-crds directory.
```

Comment:
maybe we can have a different directive for `kf-training-operator` that contains all the manifests?

Reply:
Yes, we can do that, we need it and I wi explain it below.